admin-guide.md

```diff
@@ -270,3 +277,19 @@ will be captured. Then `"metrics_calculate"` will be called with the
 `"output_metrics"` as the first argument and the file generated in the previous
 step as the second argument. If it exits with a non-zero return code, then the
 test is allowed to pass.
+
+## Testing Víðarr Locally
+
+To test Víðarr locally, ensure that you satisfy these requirements:
+
+- Make sure the .sh file created from the instructions provided in [this step](https://github.com/oicr-gsi/vidarr/blob/master/admin-guide.md#creating-a-development-environment) points to the jars you created [here](https://github.com/oicr-gsi/vidarr/blob/master/admin-guide.md#installing-v%C3%AD%C3%B0arr-on-a-linux-instance).
+- Use the JSON config file generated during that step as the configuration argument in your command.
+  - Update the `"url"` and `"cromwellUrl"` blocks within the JSON file to the secret URL provided in [this file](https://bitbucket.oicr.on.ca/projects/GSI/repos/infrastructure/browse/vidarr/development/development.vidarrconfig.in).
```

Comment:
Bitbucket repos are private so shouldn't reference them in vidarr (a public repo).
If this testing strategy is only relevant to our team can potentially be better served as a wiki page rather than made public

Reply:
Okay yes makes sense, I can make a wiki as well. I just thought you also need internal credentials to access the Bitbucket repos so even though the link is public, you can only access the contents itself with the necessary authentication.

Reply:
You're right, it's not accessible so there's no security risk. Just better not to have "broken" links that people cannot use

Reply:
Probably reword this step to be more generally applicable - Update the 'url' block to a URL representing your environment, and the cromwellUrl to your cromwell server's REST API. If anyone outside OICR uses vidarr, they will know these details about their own environment.